app/lib/definitions.ts

```diff
@@ -104,4 +104,24 @@ export type ProfileData = {
     createdAt: Date
 }
 
-// #endregion Profile
\ No newline at end of file
+// #endregion Profile
+
+// #region Comment
+
+export type IComment = {
+    id: string,
+    text: string,
+    createdAt: Date,
+    modifiedAt: Date,
+    Owner: {
```

Comment:
It's a bit of a nit picking, but `Owner` could be its own type too

Reply:
In general, it's a good practice to create new types for these nested objects/types